static/bidder-params/openweb.json

```diff
@@ -2,25 +2,34 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "OpenWeb Adapter Params",
   "description": "A schema which validates params accepted by the OpenWeb adapter",
-
   "type": "object",
   "properties": {
     "placementId": {
-      "type": "integer",
-      "description": "An ID which identifies this placement of the impression"
-    },
-    "siteId": {
-      "type": "integer",
-      "description": "An ID which identifies the site selling the impression"
+      "type": "string",
+      "description": "An ID which identifies this placement of the impression",
+      "minLength": 1
     },
     "aid": {
       "type": "integer",
-      "description": "An ID which identifies the channel"
+      "description": "Deprecated: An ID which identifies the channel"
     },
-    "bidFloor": {
-      "type": "number",
-      "description": "BidFloor, US Dollars"
+    "org": {
+      "type": "string",
+      "description": "The organization ID.",
+      "minLength": 1
     }
   },
-  "required": ["aid"]
+  "required": [ "placementId" ],
+  "oneOf": [
+    {
+      "required": [
+        "aid"
+      ]
+    },
+    {
+      "required": [
+        "org"
+      ]
+    }
+  ]
```

Comment:
should update/add json tests
- to have request with required param placementId
- to have request with aid and org is absent
- to have request with org and aid is absent

Reply:
I added 1 test with the `placementId` absent: `missing-placement-id.json`
I changed the `simple-video.json` test case to send org ID as aid instead of org.
The `simple-banner.json` test case still sends only org (aid is absent).